pgvectorscale/src/access_method/options.rs

```diff
@@ -184,7 +185,10 @@ pub unsafe fn init() {
         "Storage layout: either memory_optimized, io_optimized, or plain".as_pg_cstr(),
         super::storage::DEFAULT_STORAGE_TYPE_STR.as_pg_cstr(),
         Some(validate_storage_layout),
-        pg_sys::AccessExclusiveLock as pg_sys::LOCKMODE,
+        #[cfg(any(feature = "pg13", feature = "pg14", feature = "pg15", feature = "pg16"))]
+        {
+            pg_sys::AccessExclusiveLock as pg_sys::LOCKMODE
+        },
```

Comment:
Only relevant for PG12 (same with all the other changes in this file)

Reply:
fixed